client/ayon_deadline/plugins/publish/blender/submit_blender_deadline.py

```diff
@@ -24,7 +24,66 @@ class BlenderSubmitDeadline(abstract_submit_deadline.AbstractSubmitDeadline,
     families = ["render"]  # TODO this should be farm specific as render.farm
     settings_category = "deadline"
 
-    def get_job_info(self, job_info=None):
+    def process(self, instance):
+        if not instance.data.get("farm"):
+            self.log.debug("Render on farm is disabled. "
+                           "Skipping deadline submission.")
+            return
+
+        # Always set instance output directory to the expected
+        expected_files = instance.data["expectedFiles"]
+        if not expected_files:
+            raise RuntimeError("No Render Elements found!")
```

Comment:
This should use `PublishError`...

Reply:
Fixed with https://github.com/ynput/ayon-deadline/pull/187/commits/c117987a94ba9002fd5c90e2ba55aefdb6d05709